### ISSUE TITLE
Login failed!

### ISSUE THREAD

**Issue by cphubs** [RIYIC/system_user#1]
Hi 

I create new user with this json file

{ 
   "system":{
   "users": {
     "lshell": [
       {
         "username": "user1",
         "password": "user1",
        "group": "users",
         "shell_options": {
           "intro" : "== Cloud ==\nWelcome to shell\nType '?' or 'help' to get the list of allowed commands",
           "path"  : ["/home/user1"]
         }
       }
     ]
   }
 },
  "run_list": [  
  "recipe[system_user::with_lshell]"
  ]
}

but when login into this user with putty exit imediatelly

thanks

**Comment by alambike** [comment]
Hi,
You must scape "\n" in a double quotes string to full string arrives right to chef template. So "intro" option must look like:

```
"intro" : "== Cloud ==\\nWelcome to shell\\nType '?' or 'help' to get the list of allowed commands",
```

Anyway, there is a bug in shell_lshell  cookbook (fixed now in 0.1.2 version) because use of single quotes string in lshell config template .

**Comment by cphubs** [comment]
Ok Thanks
